src/js/common/schemaValidatedComponent.js

```diff
@@ -92,211 +92,7 @@ var fluid  = fluid  || require("infusion");
     fluid.defaults("fluid.schema.component", {
         gradeNames: ["fluid.component", "fluid.contextAware"],
         schema: {
-            "$schema": "fss-v7-full#",
-            "definitions": {
```

Comment:
Shouldn't this material have gone into another grade rather than being axed completely?

Reply:
No, none of those definitions or properties are necessary to establish your own contract about what options you expect.  All of the rules there and in the model validated component were trying to ensure that basic component material like listeners, et cetera were defined correctly.

Reply:
Well that is still valuable material to have expressed somehow, even if it is provisional - could it be retained and tested in a grade labelled as experimental or so?

Reply:
Yes, I can reconstitute it as an experimental grade with its own tests.

Reply:
OK, I made a pass at it, this should be ready for review again.